roles/cfg_openwrt/templates/corerouter/bird.conf.j2

```diff
@@ -45,14 +60,12 @@ protocol babel {
   {% set default_mesh_metric = hostvars[inventory_hostname].get('mesh_metric_adhoc_' ~ nw.get('mesh_radio'), mesh_metric_default_in) %} 
```

Comment:
the comment should be moved up and the line with `set default_mesh_metric` should be removed as it does not do anything here anymore